woocommerce/woocommerce-ios
# Orders: Wiring defaultStoreID Session property

### Details:
This PR removes the hardcoded siteID present in OrdersViewController. 

### Testing:
1. Fresh install WC
2. Log into your account
3. Verify that the Orders List displays the entries associated to the selected site.
